lib/tessel/deploy.js

```diff
@@ -225,8 +225,11 @@ actions.runScript = function(t, filepath, entryPoint) {
           resolve();
         }
 
-        // Once we get a SIGINT from the console, stop the script
-        process.once('SIGINT', scriptStopped);
+        // Once we get a SIGINT from the console
+        process.once('SIGINT', function() {
+          // pass the SIGINT to the remote process
+          remoteProcess.stdout.signal('SIGINT');
+        });
```

Comment:
Also, we can just move the contents of `scriptStopped` to the `remoteProcess.once('close')` event because I don't think any other functions call it.